### PR TITLE
Update comment of ~ updjud

### PR DESCRIPTION
**Pull request by avekens**
Remark about "coproduct" added (as discussed in  PR #3406).

### CODE REVIEW

**Review by icecream17 — APPROVED** [review]
the wikipedia article additionally confirms the idea, as someone who has previously never seen the term

**Review by benjub — APPROVED** [review]
I think the wikipedia link is unnecessary.

**Comment by avekens** [comment]
To be more professional/reputable, I provided a bibliographic reference to the book of Adamek et al. in addition to the Wikipedia link. I think a bibliograohic reference is kind of old fashioned - nowadays, additional information must be (at most) one click away...

File: set.mm [hunk 1]
```diff
@@ -94995,8 +94995,13 @@ of wff (meta)variables, see ~ htalem .  (Contributed by NM,
     $d G h k x $.  $d G k x y z $.  $d h k ph x $.  $d ph x y z $.
     updjud.a $e |- ( ph -> A e. V ) $.
     updjud.b $e |- ( ph -> B e. W ) $.
-    $( Universal property of the disjoint union.  (Proposed by BJ,
-       25-Jun-2022.)  (Contributed by AV, 28-Jun-2022.) $)
+    $( Universal property of the disjoint union.  This theorem shows that the
+       disjoint union, together with the left and right injections ~ df-inl and
+       ~ df-inr , is the _coproduct_ in the category of sets, see Wikipedia
+       "Coproduct", ~ https://en.wikipedia.org/wiki/Coproduct (25-Aug-2023).
+       This is a special case of example 1 of coproducts in section 10.67 of
```
Comment (benjub) [inline set.mm:95002]:
```suggestion
       This is a special case of Example 1 of coproducts in Section 10.67 of
```

**Review by benjub — APPROVED** [review]

**Review by benjub — APPROVED** [review]

**Pull request merged**